testing/unittests/jlibtests.cpp

```diff
@@ -2448,7 +2447,7 @@ class AtomicTimingTest : public CppUnit::TestFixture
     {};
 
     const unsigned numIterations = 1000000;
-    const unsigned numCores = getAffinityCpus();
+    const unsigned numCores = std::max(getAffinityCpus(), 16U);
```

Comment:
Limit the number to something sensible - otherwise it can take forever if you have lots of cores....